### PR TITLE
feat: compatible with assertions

### LINKED ISSUE #30: allow to customize `keyword`
Will be great to allow setup `keyword` on different names, in webpack we need support `assert` and `with` together for compatibility, yes we can use old `acorn-import-asse` package, but there is a problem - no bugs fixes in `acorn-import-assertions`, because it was renamed

/cc @xtuc

### PR DESCRIPTION
**Pull request by ahabhgk**
resolve #30

### CODE REVIEW

**Review by xtuc — APPROVED** [review]
Tests are passing locally.

Nice one, thanks!

**Comment by alexander-akait** [comment]
@ahabhgk Thank you :heart:

File: src/index.js [hunk 1]
```diff
@@ -32,12 +48,42 @@ export function importAttributes(Parser) {
...
 
+    _matchKeywordToken() {
+      return (isWithOrAssertKeyword && (this.type === this.withToken || this.type === this.assertToken))
+        || (isWithKeyword && this.type === this.withToken)
+        || (isAssertKeyword && this.type === this.assertToken)
+    }
+
+    _getProperty() {
+      if (isWithOrAssertKeyword) {
+        return this.type === this.withToken ? "attributes" : "assertions";
```
Comment (alexander-akait) [inline src/index.js:59]: @ahabhgk I think there is a bug, logs:
```
console.log(this.type);
console.log(this.withToken);
```

```
 TokenType {
        label: ';',
        keyword: undefined,
        beforeExpr: true,
        startsExpr: false,
        isLoop: false,
        isAssign: false,
        prefix: false,
        postfix: false,
        binop: null,
        updateContext: null
      }

      at Parser._getProperty (node_modules/acorn-import-attributes/lib/index.js:68:13)

    console.log
      TokenType {
        label: 'with',
        keyword: undefined,
        beforeExpr: false,
        startsExpr: false,
        isLoop: false,
        isAssign: false,
        prefix: false,
        postfix: false,
        binop: null,
        updateContext: null
      }

      at Parser._getProperty (node_modules/acorn-import-attributes/lib/index.js:69:13)

```

So it is always `assertions`

Comment (ahabhgk) [reply src/index.js:59]: Didn't notice there is a `this.next()`.. https://github.com/xtuc/acorn-import-attributes/pull/33 should fix it

Comment (xtuc) [reply src/index.js:59]: merged and published `acorn-import-attributes@1.9.5`

**Pull request merged**